src/appleseed/renderer/modeling/frame/frame.cpp

```diff
@@ -44,11 +44,12 @@
 
 // appleseed.foundation headers.
 #include "foundation/image/color.h"
-#include "foundation/image/exrimagefilewriter.h"
+#include "foundation/image/drawing.h"
```

Comment:
Remove that #include statement (bad merge?)

Reply:
Yep, bag merge.
Done